### PR TITLE
Create an event type

### PR DESCRIPTION
**Pull request by erichardson30**
## Why?
As a developer I need to create an event type.

## What changed?
* Added function in events module to create an event type
* Updated tests

### CODE REVIEW

File: src/events/index.js [hunk 1]
```diff
@@ -316,6 +317,64 @@ class Events {
...
+   * Method: POST
+   *
+   * @param {Object} eventType
+   * @param {string} eventType.name
+   * @param {string} eventType.description
+   * @param {integer} [eventType.level]
+   * @param {string} eventType.clientId
+   * @param {string} eventType.slug
+   * @param {boolean} eventType.isRealtimeEnabled
+   * @param {boolean} eventType.isOngoingEvent
```
Comment (greg5green) [inline src/events/index.js]: Can you add a little more info about these keys? There isn't a ton in the regular typedef either

File: src/events/index.js [hunk 2]
```diff
@@ -316,6 +317,64 @@ class Events {
...
+  /**
+   * Creates a new event type
+   *
+   * API Endpoint: '/types'
+   * Method: POST
+   *
+   * @param {Object} eventType
+   * @param {string} eventType.name
+   * @param {string} eventType.description
+   * @param {integer} [eventType.level]
```
Comment (greg5green) [inline src/events/index.js]: We've been using `number` for the type since thats the only JS type

File: src/events/index.spec.js [hunk 3]
```diff
@@ -525,4 +525,99 @@ describe('Events', function() {
...
+      it('creates a new event type', function() {
+        expect(request.post).to.be.deep.calledWith(
+          `${expectedHost}/types`,
+          eventTypeToServerAfterFormat
+        );
+      });
+
+      it('formats the returned object', function() {
+        return promise.then(() => {
+          expect(toCamelCase).to.be.deep.calledWith(
```
Comment (greg5green) [inline src/events/index.spec.js]: Why would all these tests be `deep`?

Comment (erichardson30) [reply src/events/index.spec.js]: Keeping consistent with all the other `create` tests.

Comment (erichardson30) [reply src/events/index.spec.js]: Updated

**Pull request merged**